musescore@3.5.2.311459983: download via aria2 failed

Seems like musescore changed release files naming and the correct file name should be `MuseScore-3.5.2.311459983-x86_64.msi`



```
 scoop update *
musescore: 3.5.0.13199 -> 3.5.2.311459983
ocenaudio: 3.9.3 -> 3.9.4
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Updating 2 outdated apps:
Updating 'musescore' (3.5.0.13199 -> 3.5.2.311459983)
Downloading new version
Starting download with aria2 ...
Download: 10/20 09:00:03 [ERROR] CUID#8 - Download aborted. URI=https://github.com/musescore/MuseScore/releases/download/v3.5.2/MuseScore-3.5.2-x86_64.msi
Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/musescore/MuseScore/releases/download/v3.5.2/MuseScore-3.5.2-x86_64.msi
Download:   -> [HttpSkipResponseCommand.cc:219] errorCode=3 Resource not found
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 2852c7|ERR |       0B/s|C:/Users/NNL/scoop/cache/musescore#3.5.2.311459983#https_github.com_musescore_MuseScore_releases_download_v3.5.2_MuseScore-3.5.2-x86_64.msi
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.
ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/musescore/MuseScore/releases/download/v3.5.2/MuseScore-3.5.2-x86_64.msi
    referer=https://github.com/musescore/MuseScore/releases/download/v3.5.2/
    dir=C:\Users\NNL\scoop\cache
    out=musescore#3.5.2.311459983#https_github.com_musescore_MuseScore_releases_download_v3.5.2_MuseScore-3.5.2-x86_64.msi

ERROR & 'C:\Users\NNL\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\NNL\scoop\cache\musescore.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/5.1 (Windows NT 10.0; Win64; x64; Desktop)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=26160 --continue

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=musescore%403.5.2.311459983%3a+download+via+aria2+failed
```

--- a/bucket/musescore.json
+++ b/bucket/musescore.json
@@ -42,7 +42,7 @@
                 }
             },
             "32bit": {
-                "url": "https://github.com/musescore/MuseScore/releases/download/$matchUrl.msi",
+                "url": "https://github.com/musescore/MuseScore/releases/download/v$matchHead/MuseScore-$version-x86.msi",
                 "hash": {
                     "url": "https://musescore.org/en/download/musescore-32bit.msi",
                     "regex": ">SHA256 Checksum:\\s+$sha256</"

--- a/bucket/musescore.json
+++ b/bucket/musescore.json
@@ -35,7 +35,7 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/musescore/MuseScore/releases/download/$matchUrl_64.msi",
+                "url": "https://github.com/musescore/MuseScore/releases/download/v$matchHead/MuseScore-$version-x86_64.msi",
                 "hash": {
                     "url": "https://musescore.org/en/download/musescore.msi",
                     "regex": ">SHA256 Checksum:\\s+$sha256</"

--- a/bucket/musescore.json
+++ b/bucket/musescore.json
@@ -30,7 +30,7 @@
     ],
     "checkver": {
         "url": "https://sparkle.musescore.org/stable/3/win/appcast.xml",
-        "regex": "download/(?<url>[^\"]+)_64.msi\"\\s+sparkle:version=\"(?<version>[\\d.]+)\""
+        "regex": "sparkle:version=\"([\\d.]+)\""
     },
     "autoupdate": {
         "architecture": {

--- a/bucket/musescore.json
+++ b/bucket/musescore.json
@@ -5,11 +5,11 @@
     "license": "GPL-2.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/musescore/MuseScore/releases/download/v3.5.2/MuseScore-3.5.2-x86_64.msi",
+            "url": "https://github.com/musescore/MuseScore/releases/download/v3.5.2/MuseScore-3.5.2.311459983-x86_64.msi",
             "hash": "0e3fb8e2c5d0aadd9a8981767794837c8494b2e317882f7c1542f2091a62e0c7"
         },
         "32bit": {
-            "url": "https://github.com/musescore/MuseScore/releases/download/v3.5.2/MuseScore-3.5.2-x86.msi",
+            "url": "https://github.com/musescore/MuseScore/releases/download/v3.5.2/MuseScore-3.5.2.311459983-x86.msi",
             "hash": "7434a8c4ff21e975bc1aac5542741e38704d47c3310f5b700dcdb2c8ba7ff57d"
         }
     },
@@ -30,19 +30,19 @@
     ],
     "checkver": {
         "url": "https://sparkle.musescore.org/stable/3/win/appcast.xml",
-        "regex": "download/v(?<short>[\\d.]+)/[^\"]+\"\\s+sparkle:version=\"(?<version>[\\d.]+)\""
+        "regex": "download/(?<url>[^\"]+)_64.msi\"\\s+sparkle:version=\"(?<version>[\\d.]+)\""
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/musescore/MuseScore/releases/download/v$matchShort/MuseScore-$matchHead-x86_64.msi",
+                "url": "https://github.com/musescore/MuseScore/releases/download/$matchUrl_64.msi",
                 "hash": {
                     "url": "https://musescore.org/en/download/musescore.msi",
                     "regex": ">SHA256 Checksum:\\s+$sha256</"
                 }
             },
             "32bit": {
-                "url": "https://github.com/musescore/MuseScore/releases/download/v$matchShort/MuseScore-$matchHead-x86.msi",
+                "url": "https://github.com/musescore/MuseScore/releases/download/$matchUrl.msi",
                 "hash": {
                     "url": "https://musescore.org/en/download/musescore-32bit.msi",
                     "regex": ">SHA256 Checksum:\\s+$sha256</"

